unime/src-tauri/Cargo.toml

```diff
@@ -38,36 +18,36 @@ crate-type = ["staticlib", "cdylib", "rlib"]
 tauri-build = { version = "=2.0.0-alpha.14", features = [] }
 
 [dependencies]
-identity-wallet = { path = "./identity_wallet" }
+tauri = { version = "=2.0.0-alpha.21", features = [
+    "protocol-asset",
+    "rustls-tls",
+] }
+
+oid4vc.workspace = true
+log.workspace = true
+serde_json.workspace = true
+did-key.workspace = true
```

Comment:
`did-key` used to be under **[dev-dependencies]**. Why is it in actual dependencies now?